src/overview/components/Overview.jsx

```diff
@@ -5,11 +5,13 @@ import * as actions from '../actions'
 import { ourState } from '../selectors'
 import ResultList from './ResultList'
 
+import styles from './Overview.css'
```

Comment:
I've created the styles in the root components directory for now since we only have three components, I'd imagine that if we wish to add more each component could be nested in its own folder with its own styles.